### DISCUSSION TITLE
Unable to generate the Test Results reports after running automation test scripts through GitHub actions.

### DISCUSSION THREAD

**Discussion by KamsaniAkhil** [community/community#42849] [Actions]
### Select Topic Area

Product Feedback

### Body

Able to run selenium java automation using yml  file but unable to get result report after build run. If I provided xml path in the yml file GitHub actions will showing existing result report only not showing Test report which I run in the GitHub actions. Kindly Help me out this.
<img width="459" alt="image" src="https://user-images.githubusercontent.com/113347025/209955796-c74856b7-9f6d-46b0-8f0b-779c2c2830c1.png">

**Comment by DeLaphante** [comment]
Struggling to design or build an advanced, stable and well structured test automation framework?

Check out & play around with my powerful and robust Selenium Web UI & API test automation framework on Github -> https://github.com/DeLaphante/CynkyAutomation

Save yourself some time and use this ready made framework for any project and get the technical support you need to succeed!

**Comment by Utsav247-crypto** [comment]
Any updates or fixes on this?
